.circleci/config.yml

```diff
@@ -1,649 +1,648 @@
-                  "attachments": [
-                    {
-                      "color": "#d83933",
-                      "fallback": "<< parameters.env >> frontend deployment has failed",
-                      "text": "<< parameters.env >> frontend deployment has failed"
-                    }
-                  ]
-                }'
-            fi
-            false
-
-  # Preview deployment, used to put up preview links in pull requests
-  preview deploy:
-    docker:
-      - image: python:3
-    steps:
-      - checkout
-      - run:
-          name: deploy to preview
-          working_directory: ~/project/bin/preview-deploy
-          command: |
-            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
-              apt-get -qq update
-              apt-get -qq install jq -y
-              pip install --quiet awscli
-              PRNUM=$(basename $CIRCLE_PULL_REQUEST)
-              URL=$(./aws.sh \
-                --API_PBKDF2_ITERATIONS "$PREVIEW_API_PBKDF2_ITERATIONS" \
-                --AWS_AMI "$PREVIEW_AWS_AMI" \
-                --AWS_REGION "$PREVIEW_AWS_REGION" \
-                --AWS_SECURITY_GROUP "$PREVIEW_AWS_SECURITY_GROUP" \
-                --AWS_SUBNET "$PREVIEW_AWS_SUBNET" \
-                --BRANCH "$CIRCLE_BRANCH" \
-                --PR_NUM "$PRNUM" \
-                --OKTA_DOMAIN "$PREVIEW_OKTA_DOMAIN" \
-                --OKTA_SERVER_ID "$PREVIEW_OKTA_SERVER_ID" \
-                --OKTA_CLIENT_ID "$PREVIEW_OKTA_CLIENT_ID" \
-                --OKTA_API_KEY "$PREVIEW_OKTA_API_KEY"
-              )
-              ./github-comment.sh "$PRNUM" "https://$URL" "$CIRCLE_SHA1"
-            else
-              echo "Not a pull request"
-            fi
-
-  # Zips up artifacts created in build steps and makes them available as
-  # CircleCI artifact downloads.
-  store artifacts:
-    docker:
-      - image: node:10
-    steps:
-      - checkout
-      - attach_workspace:
-          at: ~/project
-      - run:
-          name: create backend build artifact zip
-          working_directory: ~/project
-          command: |
-            apt-get update
-            apt-get install zip -y
-            rm -rf api/seeds/development
-            rm -rf api/seeds/test
-            rm api/seeds/development.js
-            rm api/seeds/test.js
-            rm api/seeds/shared/delete-everything.js
-            cd api
-            npm ci --only=production
-            cd ..
-            zip -r /tmp/backend.zip api/*
-            zip -r /tmp/frontend.zip web/dist/*
-      - run:
-          name: persist build URL to workspace
-          command: echo "https://${CIRCLE_BUILD_NUM}-${GH_REPO_ID}-gh.circle-artifacts.com/0" > build-url.txt
-      - persist_to_workspace:
-          root: ~/project
-          paths:
-            - ./build-url.txt
-      - store_artifacts:
-          path: /tmp/backend.zip
-          destination: backend.zip
-      - store_artifacts:
-          path: /tmp/frontend.zip
-          destination: frontend.zip
```

Comment:
Has GitHub lost its mind? I can see a line was added (based on line numbers), but why does GitHub think the whole file has changed?

Reply:
Line 596: `--AWS_AMI "$PREVIEW_AWS_AMI" \` was removed

Reply:
Yea, I reverted a few commits because of that, some weird stuff was going on.